php/utils-wp.php

```diff
@@ -220,14 +220,14 @@ function wp_get_cache_type() {
 
 	if ( ! empty( $_wp_using_ext_object_cache ) ) {
 		// Test for Memcached PECL extension memcached object cache (https://github.com/tollmanz/wordpress-memcached-backend)
-		if ( isset( $wp_object_cache->m ) && is_a( $wp_object_cache->m, 'Memcached' ) ) {
+		if ( isset( $wp_object_cache->m ) && ( $wp_object_cache->m instanceof \Memcached ) ) {
```

Comment:
Parentheses are not needed here:
```suggestion
		if ( isset( $wp_object_cache->m ) && $wp_object_cache->m instanceof \Memcached ) {
```